content/swift/concepts/sets/terms/symmetricdifference/symmetricdifference.md

```diff
@@ -0,0 +1,40 @@
+---
+Title: 'Text'
+Description: 'Returns a new set with the elements.'
+Subjects:
+  - 'Software Development'
+  - 'Mobile Development'
+  - 'Computer Science'
+Tags:
+  - 'Sets'
+  - 'Methods'
+CatalogContent:
+  - 'learn-swift'
+  - 'paths/build-ios-apps-with-swiftui'
+---
+
+The **.symmetricDifference()** method returns a new set with the elements that are either in the set or in the given sequence, but not in both.
+
+## Syntax
+
+```pseudo
+func symmetricDifference<S>(_other: S) -> Set<Element> where Element == S.Element, S:Sequence
+```
+
+Available when _Element_ conforms to _Hashable_
```

Comment:
```suggestion
Available when Element conforms to Hashable
```

In addition to removing the emphasis, can we unpack this line a little more? What makes an `Element` "hashable", exactly? Also, let's include info about what the `other` parameter is.